transports/tls/tls.go

```diff
@@ -1,28 +1,122 @@
 package tls
 
 import (
-	"net"
 	"crypto/tls"
+	"crypto/x509"
+	"io/ioutil"
+	"log"
+	"net"
+	"os"
+	"strings"
 
 	"github.com/gliderlabs/logspout/adapters/raw"
 	"github.com/gliderlabs/logspout/router"
 )
 
+const (
+	CA_PATH   = "/mnt/ca/"
+	CERT_PATH = "/mnt/cert/"
+)
+
 func init() {
 	router.AdapterTransports.Register(new(tlsTransport), "tls")
 	// convenience adapters around raw adapter
 	router.AdapterFactories.Register(rawTLSAdapter, "tls")
 }
 
+func getopt(name, dfault string) string {
+	value := os.Getenv(name)
+	if value == "" {
+		value = dfault
+	}
+	return value
+}
+
 func rawTLSAdapter(route *router.Route) (router.LogAdapter, error) {
 	route.Adapter = "raw+tls"
 	return raw.NewRawAdapter(route)
 }
 
+func getCAs(path string) *x509.CertPool {
+	cafiles, err := ioutil.ReadDir(path)
+	if err != nil {
+		return nil
+	}
+	capool := x509.NewCertPool()
+	// TODO: don't khow how to check pool is empty
+	empty := true
+	for _, f := range cafiles {
+		if !f.IsDir() {
+			cacert, err := ioutil.ReadFile(path + f.Name())
+			if err != nil {
+				log.Printf("Can't read CA certificate %v: %v", path+f.Name(), err)
+				continue
+			}
+			ok := capool.AppendCertsFromPEM(cacert)
+			if !ok {
+				log.Printf("Bad CA certificate %v", path+f.Name())
+			}
+			empty = !ok && empty
+		}
+	}
+
+	if empty {
+		capool = nil
+	}
```

Comment:
If RootCAs is nil, TLS uses the host's root CA set. If RootCAs is empty CertPool, we have empty CA set. But I could be wrong.